common/src/main/java/org/vivecraft/api/client/Tracker.java

```diff
@@ -0,0 +1,72 @@
+package org.vivecraft.api.client;
+
+import net.minecraft.client.player.LocalPlayer;
+import net.minecraft.world.entity.player.Player;
+import org.vivecraft.api.VRAPI;
+
+import javax.annotation.Nullable;
+
+/**
+ * A tracker is an object that is run for the local player during the game tick or before rendering a frame only if
+ * they are in VR. Using trackers are one of the cleanest ways to interact with Vivecraft's data; it's how Vivecraft
```

Comment:
Probably on me, but I think replacing the ";" with a "," here is the right call.
```suggestion
 * they are in VR. Using trackers are one of the cleanest ways to interact with Vivecraft's data, it's how Vivecraft
```

Reply:
also, should it be `Using trackers is one of the ...` or `Trackers are one of the ...`, the current gramar is not correct I think

Reply:
for now I changed it to `is`

Reply:
can reopen this if you think it should be the other one